x/erc20/genesis_test.go

```diff
@@ -21,6 +22,66 @@ import (
 	"github.com/Canto-Network/Canto/v7/x/erc20/types"
 )
 
+var (
+	uqstars = "ibc/13B6057538B93225F6EBACCB64574C49B2C1568C5AE6CCFE0A039D7DAC02BF29"
+	// uqstars1 and uqstars2 have same denom
+	// uqstars1 is deployed first.
+	uqstars1 = types.TokenPair{
+		Erc20Address:  "0x2C68D1d6aB986Ff4640b51e1F14C716a076E44C4",
+		Denom:         uqstars,
+		Enabled:       true,
+		ContractOwner: types.OWNER_MODULE,
+	}
+	// uqstars2 is deployed later than uqstars1.
+	uqstars2 = types.TokenPair{
+		Erc20Address:  "0xD32eB974468ed767338533842D2D4Cc90B9BAb46",
+		Denom:         uqstars,
+		Enabled:       true,
+		ContractOwner: types.OWNER_MODULE,
+	}
+	customERC20 = types.TokenPair{
+		Erc20Address:  "0xC5e00D3b04563950941f7137B5AfA3a534F0D6d6",
+		Denom:         "custom",
+		Enabled:       true,
+		ContractOwner: types.OWNER_EXTERNAL,
+	}
+
+	tokenPairs = []types.TokenPair{
+		uqstars2,
+		// even if we put uqstars1 later, it should be disabled because
+		// uqstars2 is the deployed later than uqstars1
+		uqstars1,
+		customERC20,
+	}
+	denomIdxs = []types.TokenPairDenomIndex{
+		{
+			Denom:       customERC20.Denom,
+			TokenPairId: customERC20.GetID(),
+		},
+		{
+			Denom: uqstars,
+			// denomIdx must have latest token pair id
```

Comment:
The token pair id is a hash, not a number, but currently, "latest" might be misinterpreted as numerically largest. How about modifying it as suggested below?

```suggestion
			// denomIdx must have the latest token pair id assigned
```